src/main/java/com/hankyz/ymcabooking/handler/WebDriverHandler.java

```diff
@@ -166,13 +166,11 @@ public void book() {
         boolean notFound = true;
         while (notFound) {
             driver.findElement(By.xpath(searchButtonXpathSelector)).click();
+            notFound = driver.findElements(By.id("chkBook2")) == null;
             try {
-                // sleep 3 seconds
-                Thread.sleep(3000);
-                // if found we stop
-                if(driver.findElement(By.id("chkBook2")).isDisplayed()){
-                    notFound = false;
-                }
+                // sleep 0.2 second
+                Thread.sleep(200);
+                // if found stop
```

Comment:
remove checking checkboxes from the loop

Reply:
What do you mean by removing checking checkboxes from the loop?